website/docs/cdktf/create-and-deploy/deployment-patterns.mdx

```diff
@@ -0,0 +1,175 @@
+
+Refer to [UI/VCS-Driven Run Workflow](/cloud-docs/run/ui) in the Terraform Cloud documentation for details.
+
+### GitHub Actions CI and Terraform Cloud
+
+Set the `TERRAFORM_CLOUD_TOKEN` environment variable to the [Terraform Cloud API token](https://www.terraform.io/cloud-docs/users-teams-organizations/api-tokens). This allows the CDKTF stacks `RemoteBackend` to connect to Terraform Cloud.
+
+The following example shows a GitHub Actions configuration where the CDKTF application is deployed from the `main` branch and uses Terraform Cloud. This file could be stored in `.github/workflows/cdktf-deploy.yml`.
+
+```yaml
+name: CDKTF Deployment
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: "npm"
+      - name: Install CDKTF CLI
+        run: npm install -g cdktf-cli
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.1.7
+      - name: Install dependencies
+        run: npm ci
+      # If you use providers or modules that are not pre-built, you can install them here
+      - name: Generate bindings for providers and modules
+        run: cdktf get
+      - name: Run Tests
+        run: npm test
+      - name: Deploy
+        run: cdktf deploy --auto-approve '*' # Deploys all stacks
+        env:
+          TERRAFORM_CLOUD_TOKEN: ${{ secrets.TF_API_TOKEN }}
+```
+
+To get the plan output posted as a comment on your pull request, add the following to your `.github/workflows/cdktf-diff.yml` file:
+
+```yaml
+name: CDKTF Diff
+
+on: [pull_request]
+
+jobs:
+  diff:
+    name: "Terraform CDK Diff"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.1.7
+          cli_config_credentials_token: ${{ secrets.TF_CLOUD_TOKEN }}
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - run: yarn install
+
+      # We run the plan here, you need to fill in the name of your stack
+      - name: plan
+        run: cdktf plan <stack name>
+        env:
+          TERRAFORM_CLOUD_TOKEN: ${{ secrets.TF_API_TOKEN }}
+        continue-on-error: true # We want to see the plan output even if the plan fails
+
+      # You can replace this step with any workflow you like, e.g. posting a message in a slack channel
+      - uses: actions/github-script@0.9.0
+        if: github.event_name == 'pull_request'
+        env:
+          PLAN: "terraform\n${{ steps.plan.outputs.stdout }}" # We get the output from the last step
```

Comment:
We generally want to avoid using "we" in documentation. Is this CDKTF? If it's actually Terraform, please correct me!

```suggestion
          PLAN: "terraform\n${{ steps.plan.outputs.stdout }}" # CDKTF gets the output from the last step
```